### ISSUE TITLE
Use Legacy Swift Language Version

### ISSUE THREAD

**Issue by bariseser** [CocoaPods/CocoaPods#6240]
Hi everyone. I am newbie ios and pods and get compile error.

Podfile:

```
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '8.0'
use_frameworks!
inhibit_all_warnings!

def common_stuff
  pod 'Alamofire', '~> 3.0'
  pod 'SwiftyJSON', '~> 2.0'
  pod 'JGProgressHUD'
  pod 'HanekeSwift'
  pod 'AlamofireImage', '~> 2.0'
  pod 'SSZipArchive'
  pod 'RNCryptor', '~> 4.0.0-beta'
  pod 'Fabric'
  pod 'Crashlytics'
  pod 'GRDB.swift', '~> 0.70.0'
  pod 'XCGLogger',  '~> 3.2'
  pod 'RxSwift',    '~> 2.0'
  pod 'RxCocoa',    '~> 2.0'
  pod 'RxAlamofire'
  pod 'Localize-Swift', '~> 1.1'
  pod 'SideMenu'
end

target 'nav' do
  common_stuff

  target 'navTests' do
    platform :ios, '8.0'
    inherit! :search_paths

    pod 'Quick'
    pod 'Nimble'
    pod 'RxBlocking',  '~> 2.0'
  end
end

target 'Mobiki' do
  common_stuff
end
```

I am using xcode Version 8.1 (8B62) and when build project get error

Almofire: 
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

GRBD.swift: 
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

HanekeSwift:
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

How can i solve this problems. Thanks

**Comment by flbaue** [comment]
Hi, I just had the same issue this morning. For me it worked to use the Xcode Swift converter on the Alamofire Target. No source code was changed, though some settings were.

For example the converter added ```SWIFT_VERSION = 3.0;```entries to the ```/Pods/Pods.xcodeproj/project.pbxproj```

Maybe it is compatibility problem between CocoaPods and the current Xcode?

**Comment by DanToml** [comment]
@flbaue Could you try re running `pod install`?

**Comment by flbaue** [comment]
@DanToml When I rerun ```pod install``` I have the problem again. Xcode tells me that I have to use legacy Swift for Alamofire and HanekeSwift. Also right after every start Xcode asks to convert some targets.

**Comment by bariseser** [comment]
@flbaue ı cenverted swift 3 and getting 999+ compile error :)

**Comment by flbaue** [comment]
@bariseser Also after clean & build? Right after the conversion Xcode also showed me a lot of errors but they disappeared on an actual build.

**Comment by Brett-Best** [comment]
@bariseser the podfile you included has Alamofire as version 3.0+ not including 4 or above. Alamofire added Swift 3 support in version 3. For the other pods I'd suggest trying to use the latest release.

**Comment by orta** [comment]
You cannot use multiple versions of Swift in the same project, it looks like this is trying to use multiple versions of Swift - please audit all your libraries to make sure they all support the same version.
